Cubical/ZCohomology/Base.agda

```diff
@@ -0,0 +1,51 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.ZCohomology.Base where
+
+
+open import Cubical.Data.Int.Base
+open import Cubical.Data.Nat.Base
+open import Cubical.Data.NatMinusTwo.Base
+open import Cubical.Data.Sigma.Base
+
+open import Cubical.Foundations.Pointed.Base
+
+open import Cubical.HITs.Nullification.Base
+open import Cubical.HITs.SetTruncation.Base
+open import Cubical.HITs.Sn.Base
+open import Cubical.HITs.Susp.Base
+open import Cubical.HITs.Truncation.Base
+
+private
+  variable
+    ℓ : Level
+    A : Type ℓ
+
+{-
+Definition of cohomology with cocefficients in the integers. For now, this definition uses ℕ₋₂, for instance defining H⁰ as
+coHom neg2, H¹ as coHom (suc neg2), and so on. This has so far been easier to work with than a definition relying on ℕ→ℕ₋₂.
```

Comment:
This indexing is a bit strange and confusing... Let's look at it together next week

Reply:
I agree with Anders, it seems confusing to use ℕ₋₂ here. And it will get even more confusing when you want to do things like (H^n × H^m) → H^{n + m}, because then addition will need to be translated to n + m + 2 or something like that.